client/src/components/Intro/learn-alert.tsx

```diff
@@ -22,12 +22,15 @@ const LearnAlert = ({
     <Alert variant='info' className='annual-donation-alert'>
       {value && (
         <>
+          <div className='text-center'>
+            <h2>{t('learn.donation-heading')}</h2>
+            <h3>{`${value}%`}</h3>
```

Comment:
```suggestion
            <p>{`${value}%`}</p>
```
This isn't acting as a heading.